charts/influxdb2/templates/statefulset.yaml

```diff
@@ -32,16 +32,62 @@ spec:
         {{- end }}
       containers:
         - name: {{ .Chart.Name }}
-          image: "{{ .Values.image.repository }}:v{{ .Chart.AppVersion }}"
+          image: "{{ .Values.image.repository }}:{{ default .Chart.AppVersion .Values.image.tag }}"
```

Comment:
I'd consider dropping it since it's not really following standard practices. Using `non-values` to configure default behaviour of a chart template is counter intuitive